### PR TITLE
Add prim test for elementwise ops

### PR DESCRIPTION
**Pull request by Charles-hit**
### PR types

Others
### PR changes

Others
### Describe

1. add prim op test for `elementwise_add, elementwise_sub, elementwise_mul, elementwise_div` ops.
2. add 'bfloat16' dtype for `full` static prim api.

### CODE REVIEW

File: python/paddle/fluid/tests/unittests/test_elementwise_add_op.py [hunk 1]
```diff
@@ -245,40 +306,67 @@ def init_input_output(self):
...
         self.x = np.random.rand(2, 100, 3).astype(self.dtype)
         self.y = np.random.rand(100).astype(self.dtype)
         self.out = self.x + self.y.reshape(1, 100, 1)
-        self.python_api = broadcast_wrapper(shape=[1, 100, 1])
+        self.python_api = paddle.add
 
     def init_axis(self):
         self.axis = 1
 
+    def init_check_prim(self):
```
Comment (JiabinYang) [inline python/paddle/fluid/tests/unittests/test_elementwise_add_op.py]: some comments

File: python/paddle/fluid/tests/unittests/test_elementwise_add_op.py [hunk 2]
```diff
@@ -150,6 +199,7 @@ class TestBF16ElementwiseAddOp(OpTest):
     def setUp(self):
         self.op_type = "elementwise_add"
         self.python_api = paddle.add
+        self.prim_op_type = "prim"
```
Comment (JiabinYang) [inline python/paddle/fluid/tests/unittests/test_elementwise_add_op.py:202]: bad name, is_prim_op? is_composite_op?

File: python/paddle/fluid/tests/unittests/test_elementwise_add_op.py [hunk 3]
```diff
@@ -245,40 +306,67 @@ def init_input_output(self):
         self.x = np.random.rand(100, 2, 3).astype(self.dtype)
         self.y = np.random.rand(100).astype(self.dtype)
         self.out = self.x + self.y.reshape(100, 1, 1)
-        self.python_api = broadcast_wrapper(shape=[100, 1, 1])
+        self.python_api = paddle.add
 
     def init_axis(self):
         self.axis = 0
 
+    def init_check_prim(self):
```
Comment (JiabinYang) [inline python/paddle/fluid/tests/unittests/test_elementwise_add_op.py]: need_check_prim?

File: python/paddle/fluid/tests/unittests/test_elementwise_div_op.py [hunk 4]
```diff
@@ -33,9 +33,12 @@ class ElementwiseDivOp(OpTest):
     def setUp(self):
         self.op_type = "elementwise_div"
         self.python_api = paddle.divide
+        self.prim_op_type = "prim"
```
Comment (JiabinYang) [inline python/paddle/fluid/tests/unittests/test_elementwise_div_op.py:36]: same

File: python/paddle/fluid/tests/unittests/test_elementwise_div_op.py [hunk 5]
```diff
@@ -61,6 +64,9 @@ def setUp(self):
         self.grad_x = grad_x
         self.grad_y = grad_y
 
+    def skip_cinn(self):
```
Comment (JiabinYang) [inline python/paddle/fluid/tests/unittests/test_elementwise_div_op.py]: enable_cinn

File: python/paddle/fluid/tests/unittests/test_elementwise_mul_op.py [hunk 6]
```diff
@@ -205,7 +271,7 @@ def setUp(self):
         self.init_kernel_type()
 
 
-class TestElementwiseMulOp_broadcast_2(ElementwiseMulOp):
+class TestElementwiseMulOp_broadcast_2(ElementwiseMulOp_broadcast):
```
Comment (JiabinYang) [inline python/paddle/fluid/tests/unittests/test_elementwise_mul_op.py:297]: Don't skip this?

**Review by JiabinYang — COMMENTED** [review]
some comments, OpTest suite related problem can be fixed in next pr

File: python/paddle/fluid/tests/unittests/test_elementwise_mul_op.py (hunk 6 above)
Comment (Charles-hit) [reply python/paddle/fluid/tests/unittests/test_elementwise_mul_op.py:274]: it will reopen in next pr.

Comment (Charles-hit) [reply python/paddle/fluid/tests/unittests/test_elementwise_mul_op.py:297]: done

File: python/paddle/fluid/tests/unittests/test_elementwise_div_op.py (hunk 5 above)
Comment (Charles-hit) [reply python/paddle/fluid/tests/unittests/test_elementwise_div_op.py]: done

File: python/paddle/fluid/tests/unittests/test_elementwise_div_op.py (hunk 4 above)
Comment (Charles-hit) [reply python/paddle/fluid/tests/unittests/test_elementwise_div_op.py:36]: done

File: python/paddle/fluid/tests/unittests/test_elementwise_add_op.py (hunk 3 above)
Comment (Charles-hit) [reply python/paddle/fluid/tests/unittests/test_elementwise_add_op.py]: done

File: python/paddle/fluid/tests/unittests/test_elementwise_add_op.py (hunk 2 above)
Comment (Charles-hit) [reply python/paddle/fluid/tests/unittests/test_elementwise_add_op.py:202]: it will be modified in next pr.

**Review by JiabinYang — DISMISSED** [review]
LGTM

**Review by zhangting2020 — DISMISSED** [review]
LGTM for skip_check_grad_ci

**Review by phlrain — APPROVED** [review]

**Review by zhangting2020 — APPROVED** [review]

**Pull request merged**